docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,215 @@
+# Setting Up a Cluster on AWS using Kubernetes
+You can setup a cluster in [Kubernetes](http://kubernetes.io/docs/whatisk8s/).
+
+## Prerequisites
+- [kubectl](http://kubernetes.io/docs/user-guide/prereqs/)
+- an [AWS](http://aws.amazon.com) account
+- [AWS CLI](https://aws.amazon.com/cli/)
+- [Screwdriver tables on DynamoDB](datastore-setup/)
+
+## Create your cluster
```

Comment:
kubernetes cluster? screwdriver cluster? what cluster and API are you talking about?